src/plugins/vis_type_vega/public/vega_inspector/components/inspector_data_grid.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState, useCallback, useMemo, useEffect } from 'react';
+import { EuiDataGrid, EuiDataGridSorting, EuiDataGridProps } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { VegaRuntimeData } from '../vega_adapter';
+
+const inspectorDataGridAriaLabel = i18n.translate('visTypeVega.inspector.dataGrid.areaLabel', {
+  defaultMessage: 'Data grid',
+});
+
+export const InspectorDataGrid = ({ columns, data }: VegaRuntimeData) => {
+  const [pagination, setPagination] = useState({ pageIndex: 0, pageSize: 10 });
```

Comment:
This looks like too small of a starting page size, especially since the pagination controls are at the bottom of the screen on taller screens. Would appreciate a larger default.

Reply:
changed to 15